CHANGELOG.md

```diff
@@ -9,10 +9,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added observer for audio packets sent and audio packet received metrics inside `checkAudioConnectivity` in meeting readiness checker
+
 ### Removed
 
 ### Changed
 
+- Update `checkAudioConnectivity` logic in meeting readiness checker to check audio packets sent and audio packet received metrics in addition to attendee presence flag, as part of fast join changes
```

Comment:
You can drop ", as part of fast join changes" or you can re-word this to:

"To support fast join, update `DefaultMeetingReadinessChecker`'s `checkAudioConnectivity` logic to check for audio packets sent and received metrics using `metricsDidReceive` observer in addition to attendee presence condition"

With this you can drop the above "Add observer..." changelog I think.